pkg/prm/config.go

```diff
@@ -1,9 +1,24 @@
 //nolint:structcheck,unused
 package prm
 
-import "github.com/Masterminds/semver"
+import (
+	"github.com/Masterminds/semver"
+	"github.com/spf13/viper"
+)
 
-type Config struct {
+const (
+	puppetVerKey string = "puppetVer"
+)
+
+type Config struct{}
+type ConfigI interface {
+	SetPuppetVersion(ver *semver.Version)
+}
+type ConfigData struct {
 	puppetVersion semver.Version
 	backend       BackendI
 }
```

Comment:
This should be Config, also backend should become a string as previously identified

Reply:
`BackendI` will be changed to a `string` as part of GH-15